### ISSUE TITLE
[BUG][vscode-extension] Silent crash when analyzing LWC with LWC Base Config disabled

### ISSUE THREAD

**Issue by fabiobarcelona** [forcedotcom/code-analyzer#1998]
### Have you tried to resolve this issue yourself first?

- [x] I confirm I have gone through the above steps and still have an issue to report.

### Bug Description

When analyzing LWCs with decorators with the `disable_lwc_base_config` set to `true`, the extension shows success analyzing 0 files, and doesn't report any eslint error, even though the analyzed file has several.
<img width="339" height="110" alt="Image" src="https://github.com/user-attachments/assets/3460d228-7128-4466-8214-555fb6ef9059" />

Using the Output tab in VS Code shows no error (~quick fix here: logs shows code-analyzer version as 5.9.0, whereas I'm using v5.10.0~):
<img width="1545" height="154" alt="Image" src="https://github.com/user-attachments/assets/d48392f5-8cb2-4a3b-9fc8-5fd73731983a" />

Using the CLI Command returns the actual error:
```
> sf code-analyzer run --rule-selector AppExchange --rule-selector cpd --rule-selector eslint --target "**/lwc/**/*.js" --target "**/lwc/**/*.html" --target "**/lwc/**/*.css" --config-file ./config/code-analyzer.yml --output-file reports/sfca-results-lwc.html --view detail && npm run analyze:open:lwc
-----
 »   Error: Engine 'eslint' [23:1:12.242]:
 »       When scanning file 'pathToLwcJsFile' with the eslint engine, ESLint gave the following error:
 »           Parsing error: Unexpected character '@'
```

The LWC I'm analyzing uses an `@api` decorator, which is likely causing the error.

### Output / Logs

```shell

```

### Steps To Reproduce

1. Set `disable_lwc_base_config` to `true` in code-analyzer config
2. Analyze any LWC that uses decorators with the VS Code Extension and the ESLint engine

### Expected Behavior

Either return the error as a VS Code toast notification, allow non-LWC rules only to be used with LWCs too, or notify the user that disabling the LWC Base Config is not possible when analyzing LWCs.

### Operating System

Windows 11 Pro 25H2

### Visual Studio Code Version

1.109.5

### Extension Version

1.16.0

### Salesforce CLI Version

@salesforce/cli/2.124.7 win32-x64 node-v22.17.1

### Code Analyzer CLI Plugin Version

code-analyzer 5.10.0

### Node Version

v22.17.1

### Java Version

java 21.0.8 2025-07-15 LTS

### Python Version

Python 3.14.3

### Urgency

Low

**Comment by aruntyagiTutu** [comment]
Hi @fabiobarcelona 

We have released Code Analyzer v5.10.1, which fixes the decorator parsing issue you reported.

To test the fix:
1. Update to the latest version:
sf plugins install @salesforce/plugin-code-analyzer@latest
2. Run your analysis again with disable_lwc_base_config: true in your configuration.

The issue where LWC files containing decorators (@api, @track, @wire) failed to parse should now be resolved.
Please let us know if you encounter any issues with the new version.

Thanks again for reporting this
